src/sentry/integrations/slack/utils/channel.py

```diff
@@ -0,0 +1,181 @@
+import time
```

Comment:
@mgaeta Ignoring this file for review since I believe it's split from `utils.py`. If there are changes please let me know where.